docs/platforms/apple/common/usage/in-app-frames/index.mdx

```diff
@@ -46,4 +46,4 @@ SentrySDK.start { options in
 ## Static Frameworks
 
 When using static frameworks, the frameworks end up in the main executable. Therefore, the SDK currently can't detect if a frame of the main executable
```

Comment:
```suggestion
Because static frameworks end up in the main executable, if you're using one, the Sentry SDK won't be able to detect if a frame of the main executable
```